.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, needs triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**How to Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Smartphone (please complete the following information):**
```

Comment:
I'd also ask people to include information about the version of the App they are using (and possibly the steps describing how they can check that) i.e.:

 - DDG App Version: (e.g. 5.25.0)[1]
 - Device: (e.g. Pixel 2, Samsung s10)
  - OS: (e.g. Android 10)
  - If applicable, add screenshots to help explain your problem.

[1] App version can be found in...